solver/pom.xml

```diff
@@ -35,6 +35,16 @@
             <groupId>org.evomaster</groupId>
             <artifactId>evomaster-client-java-sql</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.evomaster</groupId>
```

Comment:
add comment that this creates a circular dependency. 
in long run, we don't want this module to depend on core, if want to use it outside of EM.
fine for now, but future might need an intermediate representation  X (eg convertion from X to SqlAction would be done inside core)

Reply:
or we might move this module directly inside core... let's leave it here for now, we can think about its integration later on

Reply:
Oh, okay! Will leave a comment with that info, thanks!